src/server/pfs/cmds/cmds.go

```diff
@@ -244,6 +244,8 @@ Examples:
 
 	var all bool
 	var block bool
+	var listCommitInclude cmd.RepeatedStringArg
+	var listCommitExclude cmd.RepeatedStringArg
 	var listCommitProvenance cmd.RepeatedStringArg
 	listCommit := &cobra.Command{
 		Use:   "list-commit repo-name",
```

Comment:
This usage string is wrong now isn't it since you can't do a repo-name without a `-i` before it right? I think rather than changing it to match the behavior it might make more sense to change the behavior. What if we made it so that positional arguments are interpreted as `include` commits and you only need a flag for exclusion? This is how a number of git commands work. Also I think `-x` for exclude might be more intuitive than `-e` what do you think?